core/cautils/getter/loadpolicy.go

```diff
@@ -55,6 +55,7 @@ func (lp *LoadPolicy) GetControl(controlName string) (*reporthandling.Control, e
 		} else {
 			for _, ctrl := range framework.Controls {
 				if strings.EqualFold(ctrl.Name, controlName) || strings.EqualFold(ctrl.ControlID, controlName) {
+					ctrl := ctrl
 					control = &ctrl
 					break
```

Comment:
Notice the `break`...  
The variables will not have the same address

Reply:
ok